packages/material-ui/src/Theme5/Mui5Context.tsx

```diff
@@ -1,93 +1,121 @@
-import React from 'react';
+import React, { ComponentType } from 'react';
+import { BoxProps, ButtonProps } from '@mui/material';
```

Comment:
Won't this break with only v4?

Reply:
I don't think so, but I do not have a v4 project to test with at this time. Our v5 project works fine with this code, though.